test/system/000-setup.bats

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bats
+#
+# Copyright © 2021 – 2023 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+load 'libs/helpers'
+
+@test "test suite: Set up" {
```

Comment:
Umm... why is `000-setup.bats` part of this pull request?  It was remove in `main`.  Did something go wrong with your branch?

Reply:
I'm reviewing that now, I did the rebase and I don't know what happened but this file `000-setup.bats` was added by mistake.

Reply:
@debarshiray I think am going to remove it manually